contracts/hello_world/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "soroban-hello-world-contract"
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib"]
+doctest = false
+
+[dependencies]
+soroban-sdk = { version = "22.0.1" }
```

Comment:
I think that we could update these Cargo.toml files to rely on workspace dependencies instead. But I could also see an argument for leaving this as is, so that this contract code will work regardless of where it lives. I'm going to approve this as-is, and we can think about coming back to update this.